.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: php
+php:
+  - '7'
+  - 5.6
+  - 5.5
+install:
+  - composer self-update
+  - composer install
+script:
+  - vendor/bin/robo test
```

Comment:
Is an empty newline at the end of a yaml file a requirement as well? If it is, it's missing here.